packages/status-js/src/client/client.ts

```diff
@@ -132,7 +132,7 @@ class Client {
   }
 
   static async start(options: ClientOptions): Promise<Client> {
-    const { environment = 'test' } = options
+    // const { environment = 'development' } = options
```

Comment:
Can we remove this?

Reply:
reverted in https://github.com/status-im/status-web/pull/543/commits/678c6954ca1ed4762daba675e32fef1f1738fb4d